app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :url, presence: true, uniqueness: true
+  validates :title, presence: true
+  validates :user, presence: true
```

Comment:
- [ ] this validation is unnecessary because belongs_to already validates the presence of a user